joss-paper/paper.md

```diff
@@ -38,7 +38,9 @@ bibliography: paper.bib
+Redox flow batteries (RFBs) are seen as a promising long-duration energy storage technology for grid-scale applications. Zero-dimensional models have previously been developed to understand the electrochemical cycling behaviour of vanadium-based electrolytes in RFBs [@2018_Konig_JPOWERSOURCE; @2018_Murthy_JES; @2018_Pugach_APPENERG; @2019_Lee_JECHEMENERGCONVSTOR], where the dominant capacity fade mechanism involves crossover of active species between negolyte (negative electrolyte) and posolyte (positive electrolyte) reservoirs. The development of next-generation electrolyte chemistries, such as redox-active organic molecules (RAOMs) [@2020_Kwabi_CHEMREV], in the past decade requires new models that incorporate properties inherent to novel chemistries. It is often the case that RAOMs are sufficiently bulky so as not to experience appreciable membrane crossover in an RFB, yet, unlike vanadium-ion-based electrolytes, they can experience chemical degradation leading to capacity decay. Recent work [@2021_Modak_JES; @2022_Neyhouse_JES] has extended VRFB-based zero-dimensional models to now include the effect of simple chemical degradation mechanisms of the redox-active organics in RFBs. At each time step iteration in zero-dimensional models, the concentrations of reduced and oxidized redox-actives are updated via Coulomb counting and the open-circuit voltage (OCV) of the cell is then calculated from the Nernst equation. Species concentrations and cell current are then used to determine the ohmic, activation, and mass transport overpotentials. Summing OCV and overpotentials yields the cell voltage. The cell current for the next time step is then determined by the cell voltage, via the cycling protocol. 
+
+However, coupled degradation and/or crossover mechanisms are currently not supported by these frameworks and degradation products cannot be tracked internally. Furthermore, the organic RFB literature includes reports of mechanisms of increasing complexity such as self-discharge, dimerization, multi-step degradations, etc. And therein lies the motivation for the development of `rfbzero.py`. Already, the inclusion of new degradation mechanisms in the zero-dimensional RFB model has contributed to the understanding of self-discharge mechanisms in aqueous RFBs [@2023_Fell_JES].
```

Comment:
`And therein lies the motivation for the development of rfbzero.py.` reads a bit odd to me, maybe this could be `This motivated the development of rfbzero.py, and already the inclusion of new degradation...`